cfme/tests/networks/test_sdn_topology.py

```diff
@@ -0,0 +1,57 @@
+import pytest
+import random
+
+from cfme.cloud.provider.ec2 import EC2Provider
+from cfme.utils.wait import wait_for
+
+
+pytestmark = [pytest.mark.provider([EC2Provider], scope='module')]
```

Comment:
**Question**: Why we need ec2 but not cloud provider here?

Reply:
This test was not written by me, so I don't have the answer to your question.